grafana-plugin/src/containers/ColumnsSelectorWrapper/ColumnsModal.tsx

```diff
@@ -135,7 +135,7 @@ export const ColumnsModal: React.FC<ColumnsModalProps> = observer(
                 variant="primary"
                 onClick={onAddNewColumns}
               >
-                {isLoading ? <LoadingPlaceholder className={'loader'} text="Loading..." /> : 'Add'}
+                {isLoading ? <LoadingPlaceholder className={'loadingPlaceholder'} text="Loading..." /> : 'Add'}
```

Comment:
I see `loadingPlaceholder` is a global style, why don't you use local style like `styles.loadingPlaceholder` here?